.nvmrc

```diff
@@ -0,0 +1 @@
+v12.21.0
```

Comment:
Do we need both nvmrc and engines in package.json?
I thought having the engines is enough?